Yevhenii-Antonov/README.md

```diff
@@ -0,0 +1,43 @@
+GoogleDocs
+==========
+
+
+GoogleDocs is a web-based word processing tool provided by Google. It allows users to create, edit, and collaborate on documents online. This README.md provides an overview of GoogleDocs and its key features.
+
+## Features
+
+* Document Creation: Users can create new documents from scratch or choose from a variety of pre-designed templates.
+
+* Real-time Collaboration: GoogleDocs enables multiple users to work on the same document simultaneously. Changes made by one user are instantly visible to others, and collaboration tools such as comments and suggestions facilitate communication among collaborators.
+
+* Auto-save and Version History: Documents are automatically saved as you work, minimizing the risk of data loss. Additionally, GoogleDocs keeps a detailed version history, allowing users to revert to previous versions of a document if needed.
+
+* Formatting and Styling: Users have access to a range of formatting options, including fonts, font sizes, text alignment, colors, and more. Styles can be customized to create professional-looking documents.
+
+* Embedded Media: GoogleDocs supports the insertion of images, tables, charts, and other media to enhance the visual appeal and information within documents.
+
+* Sharing and Permissions: Documents can be easily shared with others via a link or by adding collaborators' email addresses. Permission levels, such as view-only or editing access, can be set to control who can make changes to the document.
+
+* Offline Editing: GoogleDocs provides the ability to work offline, allowing users to access and edit documents even without an internet connection. Changes made offline will sync automatically once the internet connection is restored.
+
+## Get Started
```

Comment:
Use consistent wording, either _Getting Started_ here or _Contribute_ in section after the next one. Same goes for bullet points in the **Features** section. If you're addressing a user directly "Get Started is a good point, if your goal is to describe functionality without pointing out the user persona directly Present continuous is the way.